### PR TITLE
DOC-625: Create database source

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR creates a topic to document using a collection as a stream worker source.

### CODE REVIEW

File: docs/cep/table/index.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ FROM from_stream … ;
 You can also use general store syntax:
 
 ```sql
-CREATE STORE (GLOBAL|LOCAL)? <table_name> WITH(type="database", propKey=”propVal”, … , PrimaryKey='<attribute_name>', Index='<attribute_name>')(<attribute_name> <attribute_type>, ...);
+CREATE STORE (GLOBAL|LOCAL)? <table_name> WITH(type="database", propKey=”propVal”, … , Index='<attribute_name>')(<attribute_name> <attribute_type>, ...);
```
Comment (StoyanTc) [inline docs/cep/table/index.md]: @diana-macrometa - We cannot create inline indexes for `c8db` and you should also take out `Index='<attribute_name>'...`.

**Review by StoyanTc — COMMENTED** [review]
@diana-macrometa - Why the `cli` is part of those changes?

File: docs/cep/table/index.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/cep/table/index.md]: What do you mean "inline indexes"?

Updated this line, btw.

**Comment by diana-macrometa** [comment]
> @diana-macrometa - Why the `cli` is part of those changes?

Because I saw a bunch of backslashes that looked like a copy/paste error while I was searching for examples, so I fixed them.

File: docs/cep/table/index.md (hunk 1 above)
Comment (StoyanTc) [reply docs/cep/table/index.md]: @diana-macrometa - The RDBMS allow the indexes to be created along with the table definition. Such indexes are named `inline`. However this is not valid for  `c8db`.

**Review by StoyanTc — APPROVED** [review]

**Pull request merged**